server/auth/googleStrategy.js

```diff
@@ -0,0 +1,19 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+
+passport.serializeUser((user, done) => {
+  done(null, user);
+});
+
+passport.deserializeUser((user, done) => {
+  done(null, user);
+});
+
+passport.use(new GoogleStrategy({
+  clientID: '7792000047-0ugv44fqf5ii6bfu5qmi0almqev3abf5.apps.googleusercontent.com',
+  clientSecret: 'H1o3dycdzvzXfCAg9uRwe8Aw',
```

Comment:
your clientID and clientSecret should be in .env files and in your .gitignore. You need to rebase and figure out how to remove this from github....or get new creds